src/app/public/modules/list-column-selector-action/list-column-selector-button.component.html

```diff
@@ -0,0 +1,11 @@
+<button
+  class="sky-btn sky-btn-default"
+  type="button"
+  [attr.aria-label]="'skyux_grid_columns_toolbar_button' | skyLibResources"
+  [title]="'skyux_grid_columns_button' | skyLibResources"
```

Comment:
Shouldn't title and aria-label be the same?

Reply:
Good call. I believe that `skyux_grid_columns_button` is correct as it is `Choose columns` while the other label is simply `Column`. I have updated it to be the same.